sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2125,14 +2125,6 @@ private Response<RecognizeLinkedEntitiesResultCollection> RecognizeLinkedEntitie
         /// <see href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/language-support"/>.
         /// For document length limits, maximum batch size, and supported text encoding, see
         /// <see href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits"/>.
-        /// <remarks>
-        /// <para>Method is only available for <see cref="TextAnalyticsClientOptions.ServiceVersion.V3_1"/> and up.</para>
```

Comment:
isn't this part still true?

Reply:
Good catch. Too excited about deleting stuff.

Reply:
Fixed